app/page.tsx

```diff
@@ -1,11 +1,17 @@
 'use client'
+
 import { Heading, Box, Container, useColorModeValue } from '@chakra-ui/react'
 import { PrimaryButton } from '@/components/button'
 import { Header, Footer } from '@/components/navigation'
+import { useTripsCollectionSuspenseQuery } from '@generated/api'
 
 export default function Top() {
   const bg = useColorModeValue('white', 'gray.800')
   const color = useColorModeValue('black', 'gray.300')
+  const { data } = useTripsCollectionSuspenseQuery({
```

Comment:
This diff is just a sample, So It'll be remove later 🙏

Reply:
Oh custom hooks is also generated! 👏

Reply:
Btw I have an error in my local vs code, but no error when I run the command `pnpm run lint`. Did you have the same situation? The lint shows error because of import order. 

![Screenshot 2023-11-27 at 2 47 51 PM](https://github.com/tabi-memo/frontend/assets/66394413/2b5e2505-26f8-4adc-a919-ac0a46a5e022)

Reply:
Hmm... 🤔
If you are enable to `eslint.autosave` in your VSCode setting, It can be fixed automatically.

Reply:
Please forget about that 🙏  I don't see the error anymore. Not sure what was that. Sorry for the confusion..!